.cloudflare/flathub-verified/src/worker.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  async fetch(request, env, _ctx) {
+    const url = new URL(request.url);
+
+    if (url.pathname !== "/.well-known/org.flathub.VerifiedApps.txt") {
+      return await fetch("https://zed.dev/404");
+    }
```

Comment:
Note that this condition is technically redundant, but I thought I would add it in case a request was to be addressed directly to the worker. Happy to remove if you feel this is necessary